biscuit-auth/src/datalog/origin.rs

```diff
@@ -119,3 +143,9 @@ impl<'a> FromIterator<&'a usize> for TrustedOrigins {
         Self(iter.into_iter().cloned().collect())
     }
 }
+
+impl Display for TrustedOrigins {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        self.0.fmt(f)
+    }
+}
```

Comment:
Is it required by the rest of the changes? It could indicate a confusion between the origin of a rule (the block it was defined in), and its trusted origins. Avoiding this confusion is the point of the separated Origin and TrustedOrigin types.

Reply:
not required, that was a mistake, fixed in [e3c3467](https://github.com/biscuit-auth/biscuit-rust/pull/126/commits/e3c3467c7b4b5865f8f293f80415a6e25a4b073c)

Reply:
👍